### PR TITLE
[Service Bus] Changelog for management client change

### PR DESCRIPTION
**Pull request by YijunXieMS**

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 1]
```diff
@@ -26,13 +29,17 @@
...
 
-* Add `on_lock_renew_failure` as a parameter to `AutoLockRenew.register`, taking a callback for when the lock is lost non-intentially (e.g. not via settling, shutdown, or autolockrenew duration completion)
-
-**Breaking Changes**
-
 * `AutoLockRenew.sleep_time` and `AutoLockRenew.renew_period` have been made internal as `_sleep_time` and `_renew_period` respectively, as it is not expected a user will have to interact with them.
 * `AutoLockRenew.shutdown` is now `AutoLockRenew.close` to normalize with other equivelent behaviors.
 
+* Renamed `QueueDescription`, `TopicDescription`, `SubscriptionDescription` and `RuleDescription` to `QueueProperties`, `TopicProperties`, `SubscriptionProperties`, and `RuleProperties`.
+* Renamed `QueueRuntimeInfo`, `TopicRuntimeInfo`, and `SubscriptionRuntimeInfo` to `QueueRuntimeProperties`, `TopicRuntimeProperties`, and `SubscriptionRunmtimeProperties`
```
Comment (KieranBrantnerMagee) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:36]:
```suggestion
* Renamed `QueueRuntimeInfo`, `TopicRuntimeInfo`, and `SubscriptionRuntimeInfo` to `QueueRuntimeProperties`, `TopicRuntimeProperties`, and `SubscriptionRumtimeProperties`
```

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 2]
```diff
@@ -26,13 +29,17 @@
...
+* Removed param `queue` from `create_queue`, `topic` from `create_topic`, `subscription` from `create_subscription` and `rule` from `create_rule`
+ of `ServiceBusManagementClient`. Added param `name` to them and keyword arguments for queue properties and topic properties.
+* Removed model class attributes related keyword arguments from `update_queue` and `update_topic` of `ServiceBusManagementClient`. Use the model class instance instead.
```
Comment (KieranBrantnerMagee) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:39]:
```suggestion
* Removed model class attributes related keyword arguments from `update_queue` and `update_topic` of `ServiceBusManagementClient`. This is to encourage utilizing the model class instance instead as returned from a create_*, update_*, or list_* or get_* operation to ensure it is properly populated.  Properties may still be modified.
```

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 3]
```diff
@@ -26,13 +29,17 @@
...
+* Added arguments validation when creating model class `QueueProperties`, `TopicProperties`, `SubscriptionProperties` and `RuleProperties`
```
Comment (KieranBrantnerMagee) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:40]:
```suggestion
* Model classes `QueueProperties`, `TopicProperties`, `SubscriptionProperties` and `RuleProperties` require all arguments to be present for creation.  This is to protect against lack of partial updates by requiring all properties to be specified.
```

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,9 @@
 * Added new properties to PeekedMessaged and ReceivedMessage: `enqueued_sequence_number`, `dead_letter_error_description`,
 `dead_letter_reason`, `dead_letter_source`, `delivery_count` and `expires_at_utc`. Please refer to the docstring for further information.
 * Added support for sending received messages via `ServiceBusSender.send_messages`.
+* Added `on_lock_renew_failure` as a parameter to `AutoLockRenew.register`, taking a callback for when the lock is lost non-intentially (e.g. not via settling, shutdown, or autolockrenew duration completion)
+* Added new supported value types int, float, datetime and timedelta for `CorrelationFilter.properties`.
+* Added new property `parameters` to `SqlRuleFilter` and `SqlRuleAction`
```
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:14]: `requires_preprocessing` is also new to SqlRuleAction compared to b4?

Comment (YijunXieMS) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md]: Yes. Updated.

**Pull request merged**